### PR TITLE
Fixed failed test for "Privileged users can create prometheusrule in all namespaces"

### PR DESCRIPTION
**Pull request by reetika-vyas**
Fixed failed "Privileged users can create prometheusrule in all namespaces"

### CODE REVIEW

**Comment by ryankwilliams** [comment]
Hi @reetika-vyas,

Would it be possible to update the PR subject and the commit message to reflect which test is being fixed and the reason for the fix? This way once the PR is merged, when reviewing the git history for the project, it provides contributors with more insight into what the commit is covering.

File: pkg/e2e/verify/namespace_webhook.go [hunk 1]
```diff
@@ -194,17 +194,21 @@ var _ = ginkgo.Describe(namespaceWebhookTestName, func() {
...
+					err = deletePrometheusRule(namespace, prometheusName, h)
+					Expect(err).NotTo(HaveOccurred())
 				}
 			}
 		}, viper.GetFloat64(config.Tests.PollingTimeout))
 
 		util.GinkgoIt("Non-privileged users can create prometheusrule all non-privileged namespaces", func(ctx context.Context) {
 			for _, nonPrivilegedNamespace := range NONPRIV_NAMESPACES {
-				err := e2eCreatePrometheusRule(ctx, prometheusName, nonPrivilegedNamespace, DUMMY_USER, "dedicated-admins", h)
+				err := e2eCreatePrometheusRule(ctx, prometheusName, nonPrivilegedNamespace, DUMMY_USER, "DUMMY_GROUP", h)
```
Comment (jbpratt) [inline pkg/e2e/verify/namespace_webhook.go:211]:
```suggestion
				err := e2eCreatePrometheusRule(ctx, prometheusName, nonPrivilegedNamespace, DUMMY_USER, DUMMY_GROUP, h)
```

I think you are meaning to use the `DUMMY_GROUP` variable here and not a string of "DUMMY_GROUP"

Comment (jbpratt) [reply pkg/e2e/verify/namespace_webhook.go:211]: Hey @reetika-vyas this hasn't been resolved

**Comment by jbpratt** [comment]
/close

Hey @reetika-vyas I've included the prometheusrule webhook tests in #1446 and deleted this file. Thanks for the work on these tests! Closing this as it is no longer needed.

**Pull request closed without merging**